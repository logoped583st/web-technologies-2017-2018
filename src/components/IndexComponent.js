import React from 'react';
import { Fragment } from 'react';
import SearchComponent from './SearchComponent';
import MainUserInfoComponent from './MainUserInfoComponent';
import BioComponent from './BioComponent';
import AdditionalUserInfoComponent from './AdditionalUserInfoComponent';
import OrganizationsComponent from './OrganizationsComponent';
import AllReposFind from './AllReposFind';




class IndexComponent extends React.Component {
    render() {
        return (
            <Fragment>
                <SearchComponent className="SearchComponent" />
                <MainUserInfoComponent />
                <BioComponent />
                <AdditionalUserInfoComponent />
                <OrganizationsComponent />
                <AllReposFind/>
            </Fragment>
        );
    }
}
export default IndexComponent;