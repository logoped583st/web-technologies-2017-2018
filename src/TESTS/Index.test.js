import React from 'react';
import { shallow, mount, configure } from 'enzyme';
import { shallowToJson, mountToJson } from 'enzyme-to-json';
import { BrowserRouter } from 'react-router-dom';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import IndexComponent from '../components/IndexComponent';

configure({ adapter: new Adapter() });

describe('Userpage component test', () => {
    it('Userpage component rendered succesfully', () => {
        const userFollowers = shallow(
                <IndexComponent />
        );
        expect(shallowToJson(userFollowers)).toMatchSnapshot();
    });
});