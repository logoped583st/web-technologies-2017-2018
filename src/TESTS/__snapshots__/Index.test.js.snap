// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Userpage component test Userpage component rendered succesfully 1`] = `
<React.Fragment>
  <Connect(SearchComponent)
    className="SearchComponent"
  />
  <Connect(MainUserInfoComponent) />
  <Connect(BioComponent) />
  <Connect(AdditionalUserInfoComponent) />
  <Connect(OrganizationsComponent) />
  <Connect(AllReposFind) />
</React.Fragment>
`;
